multi-node/aws/pkg/config/config.go

```diff
@@ -126,6 +127,16 @@ func (c Cluster) Config() (*Config, error) {
 	config.APIServers = fmt.Sprintf("http://%s:8080", c.ControllerIP)
 	config.SecureAPIServers = fmt.Sprintf("https://%s:443", c.ControllerIP)
 	config.APIServerEndpoint = fmt.Sprintf("https://%s", c.ExternalDNSName)
+	if config.UseCalico {
+		config.K8sNetworkPlugin = "cni"
+		config.CalicoEnable = "true"
+		// Use a Hyperkube with CNI binaries
+		config.K8sVer = config.K8sVer + "-cni"
+	}else{
+		config.CalicoEnable = "false"
```

Comment:
I was just figuring that CalicoEnable is a yaml string that gets transformed into a Go boolean. I need to have a template value that I can use unchanged as a cloud-config enable flag. I wasn't sure if a good boolean would serialize into the right format (with the right caps) for a cloud-config flag so thought it better to be explicit. I'm happy to test further if you think it's worth it?